src/Frontend/Page/Overview.hs

```diff
@@ -130,9 +130,13 @@ instance ToHtml PageIdeasInDiscussion where
                            , href_ . U.listIdeas $ IdeaLocationTopic space (topic ^. _Id)
                            ] $ do
                             img_ [ src_ . U.TopStatic $ "images" </> case topic ^. topicPhase of
-                                      PhaseRefinement _ -> "theme_aus.png"
+                                      PhaseWildIdea     -> "theme_aus.png"  -- FIXME
+                                      PhaseWildFrozen   -> "theme_aus.png"  -- FIXME
```

Comment:
This should actually never happen (I tested manually), but it does in the test suite due to inconsistent Arbitrary data.  Sigh.  Let's ignore this for now.   What we would want is this:

```
--- a/src/Frontend/Page/Overview.hs
+++ b/src/Frontend/Page/Overview.hs
@@ -16,6 +16,8 @@ module Frontend.Page.Overview
     )
 where
 
+import Control.Exception (assert)
+
 import Action
 import Frontend.Fragment.IdeaList
 import Frontend.Fragment.QuorumBar ()
@@ -130,14 +132,15 @@ instance ToHtml PageIdeasInDiscussion where
                            , href_ . U.listIdeas $ IdeaLocationTopic space (topic ^. _Id)
                            ] $ do
                             img_ [ src_ . U.TopStatic $ "images" </> case topic ^. topicPhase of
-                                      PhaseWildIdea     -> "theme_aus.png"  -- FIXME
-                                      PhaseWildFrozen   -> "theme_aus.png"  -- FIXME
                                       PhaseRefinement{} -> "theme_aus.png"
                                       PhaseRefFrozen{}  -> "theme_aus.png"
                                       PhaseJury         -> "theme_pruf.png"
                                       PhaseVoting{}     -> "theme_abs.png"
                                       PhaseVotFrozen{}  -> "theme_abs.png"
                                       PhaseResult       -> "theme_ergf.png"
+
+                                      PhaseWildIdea   -> assert False $ error "PageIdeasInDiscussion"
+                                      PhaseWildFrozen -> assert False $ error "PageIdeasInDiscussion"
                                  , class_ "theme-grid-item-image"
                                  ]
                             div_ [class_ "theme-grid-item-text"] $ do

```